typescript/test/deposit.test.ts

```diff
@@ -105,6 +107,112 @@ describe("Deposit", () => {
     testnetAddress: "2MwGJ12ZNLJX3qWqPmqLGzh3EfdN5XAEGQ8",
   }
 
+  /**
+   * Checks if script given in argument is correct
+   * @param script - script as an un-prefixed hex string.
+   * @returns void
+   */
+  function shouldReturnProperDepositScript(script: string): void {
```

Comment:
Maybe:
```suggestion
  function assertValidDepositScript(script: string): void {
```

Reply:
Fixed in ebf24cdfc368b558167a7beaa08003375ad1121f